sample_projects/LEQuad/main_avr32.cpp

```diff
@@ -96,7 +96,7 @@ int main(void)
                         board.sonar_i2cxl,      // Warning:
                         board.uart0,
                         board.spektrum_satellite,
-                        board.green_led,
+                        board.state_display_megafly_rev4_,
```

Comment:
You should also update the commented lequad constructor L152 which is there for simulation purposes.